### PR TITLE
Translate tsconfig experimental and CLI options into ja

### PR DESCRIPTION
**Pull request by Quramy**
This is a part of #220 .

I translated tsconfig reference docs under 

- https://www.typescriptlang.org/v2/en/tsconfig#Experimental_Options_6177
  - Comparison for original docs: https://github.com/microsoft/TypeScript-Website/commit/2e990c2f5bc00bb877ae46f2d1565295cd1e80e4
- https://www.typescriptlang.org/v2/en/tsconfig#Command_line_Options_6171
  - Comparison for original docs: https://github.com/microsoft/TypeScript-Website/commit/fdd0d5b72dc97758463f8ecb073aa427d079d57c

### CODE REVIEW

File: packages/tsconfig-reference/copy/ja/options/experimentalDecorators.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+display: "Experimental Decorators"
+oneline: "Enable experimental support for TC39 stage 2 decorators"
+---
+
+[デコレータの実験的なサポート](https://github.com/tc39/proposal-decorators)を有効化します。
+これはTC39の標準化プロセスではstage 2の機能です。
+
+これはTypeScriptの実装バージョンが、TC39が決定するJavaScriptの実装と異なるかもしれないということを意味しています。
```
Comment (sasurau4) [inline packages/tsconfig-reference/copy/ja/options/experimentalDecorators.md]: `Decorators are a language feature which hasn't yet been fully ratified into the JavaScript specification.` が訳し忘れられてそう 👀 

```suggestion
デコレータはJavaScriptの仕様として、いまだ完全には組み込まれていない言語機能です。
これはTypeScriptの実装バージョンが、TC39が決定するJavaScriptの実装と異なるかもしれないということを意味しています。
```

**Comment by Quramy** [comment]
@sasurau4  うわぁああ、訳し忘れ、すみませんでした :bow:

**Comment by sasurau4** [comment]
LGTM 👍

**Comment by orta** [comment]
Gonna assume that the windows fail is a GitHub Actions bug

**Pull request merged**